dts/arm/nordic/nrf52832.dtsi

```diff
@@ -90,6 +91,11 @@
 			interrupts = <1 NRF_DEFAULT_IRQ_PRIORITY>;
 			status = "okay";
 			ble-2mbps-supported;
```

Comment:
One open question is, should properties like these be moved from the `radio` node to the `bt_hci` node?